pkg/templates/charts/toggle/console/templates/console-search-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: console-search-config
+  labels:
+    app: console
+    component: 'console'
+    chart: console-chart-{{ .Values.hubconfig.hubVersion }}
+    release: console-chart
+data:
```

Comment:
@zlayne FYI, you could make the string look like json with the `|` character. Here's an example.
https://stackoverflow.com/a/61654051

Reply:
This looks much better I'll change that